mm2src/coins/eth/v2_activation.rs

```diff
@@ -427,6 +427,12 @@ impl EthCoin {
             erc20_tokens_infos: Default::default(),
             nfts_infos: Default::default(),
             platform_fee_estimator_state,
+            gas_limit: if conf["gas_limit"].is_null() {
+                Default::default()
+            } else {
+                json::from_value(conf["gas_limit"].clone())
+                    .map_to_mm(|e| EthTokenActivationError::InternalError(format!("invalid gas_limit config {}", e)))?
+            },
```

Comment:
Here and in other places, create gas_limit variable before EthCoinImpl instance with `extract_gas_limit` priv function

```rust
let gas_limit = extract_gas_limit(&conf).map_to_mm(|e| EthTokenActivationError::InternalError(format!("invalid gas_limit config {}", e)))?;
```

Reply:
fixed in [2dc40ec](https://github.com/KomodoPlatform/komodo-defi-framework/pull/2137/commits/2dc40ec59244f1f9ac72afe6001c5c8da8072472)